src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -253,6 +253,9 @@
   <data name="PowerLauncher_IgnoreHotkeysInFullScreen.Content" xml:space="preserve">
     <value>Ignore hotkeys in fullscreen mode</value>
   </data>
+  <data name="PowerLauncher_DisableDriveDetectionWarning.Content" xml:space="preserve">
+    <value>Disable drive detection warning for the indexer plugin</value>

```

Comment:
Better would be:
`Disable drive indexing warning for the file search (indexer plugin)`

Is this string localized?